### ISSUE TITLE
LiveSettingProperty is not Supported for Status bar apeared in 'Workbook front page'

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#371]
### Steps to Reproduce

1. Open "Xamarin Workbook"
2. Products window is opened.
3. Navigate to the "wpf "icon and select "create workbook".
4.Navigate to target platform Combo box and expand it
5. Select any other target platform such as 'iOS'
6. Observe the Inspect property for Status bar item
### Expected Behavior
LiveSettingProperty should be True for change of status in Status bar
### Actual Behavior
LiveSettingProperty is not Supported for change of status in Status bar

Narrator Behavior: Narrator is not announcing the change of status in Status bar 
NVDA behavior:NVDA is not announcing the change of status in Status bar 

### User Impact:
Users who rely on screenreader will be unable to know the status appearing in Status bar

### Screen-Shots: 
![livesettingproperty_status_bar_inspect](https://user-images.githubusercontent.com/22956130/40725792-e92e620a-6441-11e8-8f38-b3ba4a06529f.png)

**Video Attachments:** 
[LiveSettingProperty_Status_bar_Narrator.zip](https://github.com/Microsoft/workbooks/files/2054294/LiveSettingProperty_Status_bar_Narrator.zip)

### Environment: 
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
